lib/util.js

```diff
@@ -34,15 +44,16 @@ exports.parseFilters = json => {
 
 exports.parseBody = body => {
   const str = between(body, 'var ytInitialData =', '; \n');
-  const apiKey = between(body, 'INNERTUBE_API_KEY":"', '"') || OLD_UTIL.between(body, 'innertubeApiKey":"', '"');
-  const clientVersion = between(body, 'INNERTUBE_CONTEXT_CLIENT_VERSION":"', '"') || OLD_UTIL.between(body, 'innertube_context_client_version":"', '"');
+  const apiKey = between(body, 'INNERTUBE_API_KEY":"', '"') || between(body, 'innertubeApiKey":"', '"');
+  const clientVersion = between(body, 'INNERTUBE_CONTEXT_CLIENT_VERSION":"', '"') || between(body, 'innertube_context_client_version":"', '"');
+  DEFAULT_CONTEXT.client.clientVersion = clientVersion;
```

Comment:
i know i did it myself but kinda not happy with it...
would prefer a deep-copy for each request